dashboard/src/main/resources/webapp/wizards/add-application/add-application.js

```diff
@@ -145,17 +148,20 @@ angular.module('seacloudsDashboard.projects.addApplicationWizard', ['ngRoute', '
                 case 2:
                     $scope.applicationWizardData.topology.name = $scope.applicationWizardData.name;
                     $scope.applicationWizardData.topology.application_requirements = $scope.applicationWizardData.application_requirements;
-
+                    $scope.applicationWizardData.topology.application_requirements.workload *= 100; // Fix Issue #182
```

Comment:
@adriannieto : If you step back and then step forward, the operation is performed twice.